admin.go

```diff
@@ -973,6 +982,13 @@ func handleConfig(w http.ResponseWriter, r *http.Request) error {
 		// we could consider setting up a sync.Pool for the summed
 		// hashes to reduce GC pressure.
 		w.Header().Set("Etag", makeEtag(r.URL.Path, hash))
+		_, err = w.Write(buf.Bytes())
+		if err != nil {
+			return APIError{HTTPStatus: http.StatusBadRequest, Err: err}
```

Comment:
An `InternalServerError` would be a better error 🤔 . After all, there shouldn't be any error writing the generated response.

Reply:
That's a good point. I am replacing it to Internal Server Error.